mgnn/get_relation_emb.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
+import config_predicate as args
```

Comment:
@GingerEater, I think the config_predicate file is missing.

Reply:
Hi, this file is not used in our model and I keep it just for any eventuality. I'll delete all of useless files when we finish the whole program~

Reply:
"this file" is pointing to get_relation_emb.py

Reply:
For now, please comment out the code that you are not using and later you can remove it. :)